color/p5js/Law of simmilarity + color/index.md

```diff
@@ -3,7 +3,7 @@ layout: [p5js]
 author: [Daniele Maselli]
 author-url: https://github.com/DanieleMaselli
 title: color + similarity
-tags: [P5.js, color, "Law of "]
+tags: [P5.js, color, "Law of color+similarity "]
```

Comment:
You can't do that. Sorry. If you want you can add both laws. Like this:  

    tags: [P5.js, color, Law of Color, Law of Similarity]